astro/src/content/docs/extend/events-and-webhooks/events/identity-verified.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Identity Verified
+description: Learn about the Identity Verified event.
+section: extend
+subcategory: events and webhooks
+tertcategory: events
+---
+import APIField from 'src/components/api/APIField.astro';
+import Aside from "src/components/Aside.astro";
+import Event from 'src/content/docs/extend/events-and-webhooks/events/_event.astro';
+import EventBody from 'src/content/docs/extend/events-and-webhooks/events/_event-body.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+export const eventType = 'identity.verified';
+
+<Event description="This event is generated when a user verifies an identity, which could be an email address or phone number."
```

Comment:
this `Event` component outputs some weird stuff about historic versions, not sure it makes sense to fix now.

```
This is a tenant scoped event.

It can be sent to all tenants or to one or more specified tenants. However, those tenants will not be sent events for other tenants, but only events related to themselves.

Prior to version 1.37.0 this was a tenant scoped event. This event will be sent to all tenants that are listening, but will contain a tenantId to allow for filtering.

In version 1.37.0 and later this is also a tenant scoped event. It can be sent to all tenants or to one or more specified tenants. However, those tenants will not be sent events for other tenants, but only events related to themselves.


```

Reply:
I chose option 2 - 361db8a9f72555e1d86af1904a2106f0bf989965